website/ref/builtin.md

```diff
@@ -96,20 +96,50 @@ but when the current directory is empty, `count` will wait for inputs. Hence it
 is required to put the input in a list: `count [*]` unambiguously supplies input
 in the argument, even if there is no file.
 
-## Numerical Commands
-
-Commands that operate on numbers are quite flexible about the format of the
-numbers. Integers can be specified as decimals (e.g. `233`) or hexadecimals
-(e.g. `0xE9`) and floating-point numbers can be specified using the scientific
-notation (e.g. `2.33e2`). These are different strings, but equal when considered
-as commands.
-
-Elvish has no special syntax or data type for numbers. Instead, they are just
-strings. For this reason, builtin commands for strings and numbers are
-completely separate. For instance, the numerical equality command is `==`, while
-the string equality command is `==s`. Another example is the `+` builtin, which
-only operates on numbers and does not function as a string concatenation
-commands.
+## Commands that Operate On Numbers
+
+Commands that operate on numbers are quite flexible about the
+format of those numbers. See the discussion of the [number data
+type](./language.html#number).
+
+Because numbers are normally specified as strings, rather than as
+an explicit `float64` data type, some builtin commands have variants
+intended to operate on strings or numbers exclusively. For instance, the
+numerical equality command is `==`, while the string equality command is
+`==s`. Another example is the `+` builtin, which only operates on numbers
+and does not function as a string concatenation command. Consider these
+examples:
+
+An explicit `float64` data type is created using this syntax: `(float64
```

Comment:
The paragraph and the next paragraph belong to the number type section in language.md; it is the discussion of how builtin commands work that belong to this doc.

Sorry for the back and forth; I hope it's clear now.

Reply:
Gah! Sorry, as you can see from the `... examples:` at line 111 I intended to do that since the examples are now two paragraphs later. Not sure what happened. Please take another look.